### PR TITLE
[IMP] account: redo the tour

### PR DESCRIPTION
**Pull request by william-andre**
The tour has been updated
Task [2306134](https://www.odoo.com/web#active_id=2306134&cids=1&id=2306134&model=project.task&menu_id=)

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account/static/src/js/tours/account.js [hunk 1]
```diff
@@ -8,20 +8,73 @@ var _t = core._t;
...
+        trigger: "div[name=partner_id]",
+        content: _t("Select a client.<br>Type the name of you client here. A lot of known companies are already prefilled!<br>Just start typing. If you see it in the list, click on it. Otherwise click on <b>Create and Edit</b>."),
+    }, {
+        trigger: "input[name=street]",
+        content: _t("Fill in the relevant fields needed to invoice your customer like the address and the tax information."),
+    }, {
+        trigger: "button.btn-primary",
+        content: _t("Once everything is set, you are good to continue. You will be able to edit this later in the <b>Customers</b> menu."),
+    }, {
+        trigger: "div[name=invoice_line_ids] .o_field_x2many_list_row_add a:not([data-context])",
```
[the remark this answers is not in the document]
Comment (william-andre) [reply addons/account/static/src/js/tours/account.js:52]: I tested it manually with
* Google Chrome 81.0.4044.113
* Mozilla Firefox 75.0

And with HeadlessChrome (same version as above) to do the python test; all worked.

File: addons/web_tour/static/src/js/tour_manager.js [hunk 2]
```diff
@@ -368,7 +368,7 @@ return core.Class.extend(mixins.EventDispatcherMixin, ServicesMixin, {
         tour.current_step += (inc !== undefined ? inc : 1);
         if (this.running_tour !== tour_name) {
             var index = _.findIndex(tour.steps.slice(tour.current_step), function (tip) {
-                return !tip.auto;
+                return !tip.auto || true;
```
[the remark this answers is not in the document]
Comment (william-andre) [reply addons/web_tour/static/src/js/tour_manager.js]: Was just to test something, not to keep

**Comment by oco-odoo** [comment]
@robodoo r+

**Comment by Xavier-Do** [comment]
@robodoo retry

**Comment by csnauwaert** [comment]
@robodoo r+

**Pull request closed without merging**